lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -147,7 +151,7 @@ assembleApkDebug:
     - ./gradlew assembleDebug
   artifacts:
     paths:
-      - app/build/outputs/apk/debug
+      - app/build/outputs/apk
```

Comment:
Shouldn't this be DEPLOY_DEBUG_APK_PATH? Not every repo puts its app code in the 'app' directory